### PR TITLE
[installer]: add namespace to validate cluster command

### LINKED ISSUE #7497: [Installer]: validate cluster should accept a namespace
The `validate cluster` call does not currently accept a namespace. It should accept a namespace via a `-n/--namespace` flag to run tests in that namespace.

Furthermore, it should check for the existence of the desired namespace

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

Adds a namespace flag to the validate cluster command. This updates the validation checks to receive the namespace from the flag.

A new check to ensure the existence of the namespace is added. The Installer does not create the namespace for reasons discussed [here](https://github.com/gitpod-io/gitpod/tree/main/installer#how-can-i-install-to-a-kubernetes-namespace)

Finally, have renamed the command file `validate-cluster.go` to `validate_cluster.go` to maintain consistency in naming convention and fixed a couple of nits in `checks.go`

## Related Issue(s)

Fixes #7497

## Release Notes

```release-note
[installer]: add namespace to validate cluster command
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mrsimonemms** after the PR has been reviewed.

Associated issue: *#7497*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: installer/pkg/cluster/checks.go
Comment (MrSimonEmms) [inline installer/pkg/cluster/checks.go:81]: Fixed nit

File: installer/pkg/cluster/checks.go
Comment (MrSimonEmms) [inline installer/pkg/cluster/checks.go:71]: Fixed nit

**Review by corneliusludmann — APPROVED** [review]
/lgtm
/approve

Code looks code.

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 23c5eae892545910340f6881ff45123331d06280</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7801#pullrequestreview-863559878" title="Approved">corneliusludmann</a>*

Associated issue: *#7497*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [corneliusludmann]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
